code-of-conduct.md

```diff
@@ -31,6 +31,9 @@ Examples of unacceptable behavior by participants include:
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
+This list is not meant to be exhaustive, but merely to enumerate some common
+kinds of behaviors.
```

Comment:
I believe that @pitrou's remark was about the list in the Our Pledge section. Here it's already clear that it's not exhaustive because those are "Examples".

Reply:
Indeed. I still find that "Pledge" problematic, but hey.

Reply:
Thanks for the review @ogrisel, I misread @pitrou's original comment. I moved the statement about the list being non-exhaustive to the "Our Pledge" section.